diff --git a/1-Call-MSGraph/daemon-console/Program.cs b/1-Call-MSGraph/daemon-console/Program.cs
index eee22e8..abce09d 100644
--- ./1-Call-MSGraph/daemon-console/Program.cs
+++ ./1-Call-MSGraph/daemon-console/Program.cs
@@ -51,6 +51,7 @@ namespace daemon_console
 
             if (isUsingClientSecret)
             {
+                Console.WriteLine("isUsingClientSecret --> " + isUsingClientSecret);
                 // Even if this is a console application here, a daemon application is a confidential client application
                 app = ConfidentialClientApplicationBuilder.Create(config.ClientId)
                       .WithClientSecret(config.ClientSecret)
@@ -61,12 +62,15 @@ namespace daemon_console
             else
             {
                 ICertificateLoader certificateLoader = new DefaultCertificateLoader();
+                Console.WriteLine("certificateLoader  --> 1 " + certificateLoader);
                 certificateLoader.LoadIfNeeded(config.Certificate);
+                Console.WriteLine("certificateLoader  --> 2 " + certificateLoader);
 
                 app = ConfidentialClientApplicationBuilder.Create(config.ClientId)
                       .WithCertificate(config.Certificate.Certificate)
                       .WithAuthority(new Uri(config.Authority))
                       .Build();
+                Console.WriteLine("certificateLoader  --> 3 " + certificateLoader);
             }
 
             app.AddInMemoryTokenCache();
diff --git a/1-Call-MSGraph/daemon-console/appsettings.json b/1-Call-MSGraph/daemon-console/appsettings.json
index 4af0317..486892b 100644
--- ./1-Call-MSGraph/daemon-console/appsettings.json
+++ ./1-Call-MSGraph/daemon-console/appsettings.json
@@ -1,13 +1,13 @@
 {
   "Instance": "https://login.microsoftonline.com/{0}",
   "ApiUrl": "https://graph.microsoft.com/",
-  "Tenant": "[Enter here the tenantID or domain name for your Azure AD tenant]",
-  "ClientId": "[Enter here the ClientId for your application]",
-  "ClientSecret": "[Enter here a client secret for your application]",
+  "Tenant": "fa3d34bc-81d6-4a79-ade7-175d3c33c77f",
+  "ClientId": "cf58a81b-bfc5-4942-9f5e-9cdc8d1d119d",
+  "ClientSecret": "Xzf8Q~SxY28H4UA6fd70bt39DB92xoweNC_RRc_y",
   "CertificateName": "[Or instead of client secret: Enter here the name of a certificate (from the user cert store) as registered with your application]",
   "Certificate": {
     "SourceType": "KeyVault",
-    "KeyVaultUrl": "<VaultUri>",
-    "KeyVaultCertificateName": "<CertificateName>"
+    "KeyVaultUrl": "https://cohesityprofnxj32cucakwk.vault.azure.net/",
+    "KeyVaultCertificateName": "cohesityprocert"
   }
 }
